code/modules/mob/living/blood.dm

```diff
@@ -55,17 +55,39 @@
 			if(BLOOD_VOLUME_OKAY to BLOOD_VOLUME_SAFE)
 				if(prob(5))
 					to_chat(src, "<span class='warning'>You feel [word].</span>")
+				if(prob(4.5))
+					if(prob(50))
+						Knockdown(30)
+					else
+						Stun(30)
 				adjustOxyLoss(round((BLOOD_VOLUME_NORMAL - blood_volume) * 0.01, 1))
 			if(BLOOD_VOLUME_BAD to BLOOD_VOLUME_OKAY)
 				adjustOxyLoss(round((BLOOD_VOLUME_NORMAL - blood_volume) * 0.02, 1))
 				if(prob(5))
 					blur_eyes(6)
 					to_chat(src, "<span class='warning'>You feel very [word].</span>")
-			if(BLOOD_VOLUME_SURVIVE to BLOOD_VOLUME_BAD)
+				if(prob(7.5))
+					if(prob(50))
+						Knockdown(45)
+					else
+						Stun(45)
+			if(BLOOD_VOLUME_ARREST to BLOOD_VOLUME_BAD)
 				adjustOxyLoss(5)
 				if(prob(15))
 					Unconscious(rand(20,60))
 					to_chat(src, "<span class='warning'>You feel extremely [word].</span>")
+				if(prob(17.5))
+					if(prob(50))
+						Knockdown(75)
+					else
+						Stun(75)
+			if(BLOOD_VOLUME_SURVIVE to BLOOD_VOLUME_ARREST)
+				if(prob(45)) //it doesn't instantly happen, you still have a chance to start chugging iron or hook yourself to an IV if possible
+					if(stat == CONSCIOUS)
+						visible_message("<span class='danger'>[src] clutches at [p_their()] chest as if [p_their()] heart is stopping!</span>", "<span class='userdanger'>It feels as if your heart has stopped!</span>")
+					stop_sound_channel(CHANNEL_HEARTBEAT)
+					playsound_local(src, 'sound/effects/singlebeat.ogg', 100, 0)
+					set_heartattack(TRUE)
```

Comment:
players threw a fit in my goonmed port when low blood could result in death why the fuck do you think people will like it here

Reply:
Isn't this a bit above the instant death threshold anyway? You're already taking massive oxyloss by this point